x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.tsx

```diff
@@ -74,7 +73,7 @@ export const useCasesAddToExistingCaseModal = (props: AddToExistingFlyoutProps)
 
           casesToasts.showSuccessAttach({
```

Comment:
Is it possible to adjust the text in the success toast if more than one alert has been added to a case, such as `Alerts have been added...` ?

![image](https://user-images.githubusercontent.com/7405507/165932784-528d1106-b43e-4a25-9202-ab84645e739e.png)